muselog/__init__.py

```diff
@@ -40,6 +40,10 @@ def setup_logging(root_log_level: Optional[str] = None,
             include_extra_fields=True,
             _app_name=os.environ["GRAYLOG_APP_NAME"]
         )
+        if os.environ.get("THEMUSE_ENV_LEVEL") == 2:
+            common_opts["_env"] = "production"
+        elif os.environ.get("THEMUSE_ENV_LEVEL") == 1:
```

Comment:
Can you change this to use the new variables we talked about? GRAYLOG_ENV

Reply:
done